libs/logstash-bridge/build.gradle

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+apply plugin: 'elasticsearch.build'
+
+dependencies {
+  compileOnly project(':server')
```

Comment:
@rjernst I vaguely remember us not wanting libs to depend on server but this isn't really a "lib" of traditional sorts.

Reply:
I'm glad to place it elsewhere, even in x-pack project as this does not need to be published under an OSS license. Discussions with @rjernst offline indicated that `/libs` was an appropriate starting-point.

> [The] way i would do this is have a lib, it will depend on server and painless. That lib will only live inside ES source, you can build/pull in the jars within logstash
> -- @rjernst via Slack

Reply:
This is a tricky and special case. I think we are ok to depend on server here.